### ISSUE TITLE
Towards Visually Explaining Variational Autoencoders

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#50]
# TL;DR
When anomaly detection is performed using VAE, the difference between the reconstructed image and the original image is usually taken. In this study, the visualization performance and accuracy of anomaly detection is improved by adding a branch that generates an Attention Map from the latent representation of a reconstructed image, apart from the Decoder that generates a reconstructed image from the latent representation.
![image](https://user-images.githubusercontent.com/24524018/90889183-9cc57b00-e3f2-11ea-8df0-c80a92948599.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/1911.07389

# Submission Dates(yyyy/mm/dd)

# Authors and institutions
Wenqian Liu, Runze Li, Meng Zheng, Srikrishna Karanam, Ziyan Wu, Bir Bhanu, Richard J. Radke, Octavia Camps
- Northeastern University, Boston MA 
- University of California Riverside, Riverside CA 
- Rensselaer Polytechnic Institute, Troy NY 
- United Imaging Intelligence, Cambridge MA 

# Methods

# Results

# Comments
CVPR2020
